nifi-commons/nifi-flow-encryptor/src/test/java/org/apache/nifi/flow/encryptor/StandardFlowEncryptorTest.java

```diff
@@ -89,7 +100,145 @@ public void testProcessNoEncrypted() {
         assertEquals(property, outputProperty);
     }
 
+    @Test
+    public void testProcessJson() throws JsonProcessingException {
+        final String password = "This is a bad password";
```

Comment:
Recommend changing the value to something like `String.class.getSimpleName()`.
```suggestion
        final String password = String.class.getSimpleName();
```